WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -16,6 +17,19 @@ namespace WalletWasabi.WabiSabi.Backend.Rounds;
 public partial class Arena : IWabiSabiApiRequestHandler
 {
 	public async Task<InputRegistrationResponse> RegisterInputAsync(InputRegistrationRequest request, CancellationToken cancellationToken)
+	{
+		try
+		{
+			return await RegisterInputCoreAsync(request, cancellationToken).ConfigureAwait(false);
+		}
+		catch (Exception ex) when (IsUserCheating(ex))
+		{
+			Prison.Ban(request.Input, request.RoundId);
```

Comment:
Should we log this with `LogLevel.Trace` which is disabled by default?

Reply:
Well yes, but I have to improve the logs everywhere and have another PR for that.